test/java/com/rapleaf/jack/BaseDatabaseModelTestCase.java

```diff
@@ -598,4 +599,14 @@ public void testComparable() {
     assertEquals(post3, posts.get(2));
     assertEquals(post4, posts.get(3));
   }
+
+  public void testDisableThenEnableCaching() {
```

Comment:
I don't completely understand the separation between `BaseDatabaseModelTest` and `TestAbstractDatabaseModel`–is this the right place to put this test?